derp/derphttp/derphttp_client.go

```diff
@@ -538,10 +538,10 @@ func (c *Client) tlsClient(nc net.Conn, node *tailcfg.DERPNode) *tls.Conn {
 	return tls.Client(nc, tlsConf)
 }
 
-func (c *Client) DialRegionTLS(ctx context.Context, reg *tailcfg.DERPRegion) (tlsConn *tls.Conn, connClose io.Closer, err error) {
+func (c *Client) DialRegionTLS(ctx context.Context, reg *tailcfg.DERPRegion) (tlsConn *tls.Conn, connClose io.Closer, node *tailcfg.DERPNode, err error) {
```

Comment:
can you add some docs while you're here?

It needed docs before and it really needs docs now.

Reply:
(and full sentence too, per https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences)